Homepage updates (#67)
NHSDigital/record-a-vaccination-guidance: app/index.md

---
layout: home
title: Service guidance
analytics_key: home
---

> The Record a vaccination service (RAVS) allows you to record COVID-19, Flu, Pertussis and RSV patient and staff vaccinations.  

Reviewer: Switching to a non-breaking hypen

```suggestion
The Record a vaccination service (RAVS) allows you to record COVID‑19, Flu, Pertussis and RSV patient and staff vaccinations.  
```

The service only records data. You have control over clinical judgements or decisions. 

{% from 'action-link/macro.njk' import actionLink %}

{{ actionLink({
  "text": "Log in to the pilot service (at www.ravs.england.nhs.uk)",
  "href": "https://www.ravs.england.nhs.uk"
}) }}

{{ actionLink({
  "text": "How to use the pilot service",
  "href": ("/guide/" | url)
}) }}

> {{ actionLink({
>   "text": "How to onboard organisations",
>   "href": ("/onboarding-organisations/" | url)
> }) }}

Reviewer: Suggest we remove this for now as that page is added in the other branch, and then add it back in as part of the other branch.

```suggestion
```

## Join training webinars

Register for our 1-hour training webinars from midday (12pm):  

* [Friday 16 August](https://events.teams.microsoft.com/event/0e30b1a3-6335-4a03-8b05-53c3d1047b1e@37c354b2-85b0-47f5-b222-07b48d774ee3)
* [Thursday 22 August](https://events.teams.microsoft.com/event/735f8dd2-1515-4238-a425-59e14d0195f7@37c354b2-85b0-47f5-b222-07b48d774ee3)
* [Wednesday 28 August](https://events.teams.microsoft.com/event/31f6f977-5d32-4f9d-ae06-476016dda5c7@37c354b2-85b0-47f5-b222-07b48d774ee3)
* [Tuesday 3 September](https://events.teams.microsoft.com/event/7a0b9e2e-adba-4249-a1de-793bb4b15c01@37c354b2-85b0-47f5-b222-07b48d774ee3) 
* [Wednesday 11 September](https://events.teams.microsoft.com/event/493133b7-8eb2-4d99-900e-557603d4f8a1@37c354b2-85b0-47f5-b222-07b48d774ee3)

## Help us improve

* [Feedback on the guidance or service](https://feedback.digital.nhs.uk/jfe/form/SV_ezgoupJNznAkT6m)
* [Register to participate in user research](https://feedback.digital.nhs.uk/jfe/form/SV_57HrcAOpLpt3QLY)

## Get help

> * [What to do if there's a service issue](https://guide.ravs.england.nhs.uk/service-unavailable/) 

Reviewer: ```suggestion
* [What to do if there’s a service issue](https://guide.ravs.england.nhs.uk/service-unavailable/) 
```
